02-schedule.md

```diff
@@ -21,11 +26,8 @@ AGU!
 
 ## 🍽️ Lunch! (12:00 - 1:30)
 
-:::{important}
-AGU does not provide lunch.
+Lunch will be provided in the room.
```

Comment:
Did they describe what food will be provided? Some folks may want/need to bring their own food, depending on what is provided.

Reply:
Emailed AGU about this too!